testing/src/main/java/io/stargate/it/http/RestApiv2SchemaTest.java

```diff
@@ -1186,6 +1206,17 @@ public void indexListAll() throws IOException {
         objectMapper.readValue(body, new TypeReference<List<Map<String, Object>>>() {});
 
     assertThat(data.stream().anyMatch(m -> "test_idx".equals(m.get("index_name")))).isTrue();
+
+    // but also that the other table with same name (but in diff keyspace) does not
+    body =
+        RestUtils.get(
+            authToken,
+            String.format(
+                "%s/v2/schemas/keyspaces/%s/tables/%s/indexes",
+                restUrlBase, altKeyspaceName, tableName),
+            HttpStatus.SC_OK);
+    data = objectMapper.readValue(body, new TypeReference<List<Map<String, Object>>>() {});
+    assertThat(data).isEqualTo(Arrays.asList());
```

Comment:
Curious, why `.isEqualTo(Array.asList())` vs `.isEmpty()`? Is there an advantage?

Reply:
Awesome idea! Thanks for sharing.

Reply:
Glad this was useful! I have learned many things via code reviews here, esp. constructs others use that are new to me.